stable/velero/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.0.0
 description: A Helm chart for velero
 name: velero
-version: 2.0.3
+version: 2.0.5
```

Comment:
this should be `2.0.4`

Reply:
I already bumped the version to 2.0.4 in #14378, that's why I went straight to .5 here. I can change it to .4 if you want though.

Reply:
Alright, I didn't see that PR before. Keep it as it is now.